addons/mass_mailing/views/editor_field_html.xml

```diff
@@ -7,15 +7,17 @@
 </template>
 
 <template id="assets_mail_themes_edition"> <!-- maybe to remove and convert into a field dumy with attr invisible if the template is not selected -->
-    <t t-call="web._assets_helpers"/>
+    <t t-call="web._assets_helpers">
+        <link rel="stylesheet" type="text/scss" href="/web_editor/static/src/scss/wysiwyg_variables.scss"/>
+        <link rel="stylesheet" type="text/scss" href="/mass_mailing/static/src/scss/mass_mailing.ui.scss"/>
```

Comment:
Files with css rules -> not in _assets... As I explained, only variables. Seems fine where it was before.